### ISSUE TITLE
INAV Configurator 2.1.1   - Issue with OSD configure.

### ISSUE THREAD

**Issue by JVN2000** [iNavFlight/inav-configurator#659]
Cannot configure OSD with new configurator - Blank Screen
INAV, version: 2.0.1 - MATEK F411-Wing

**Comment by gbine** [comment]
Confirm same issue with Matek F405-Wing

**Comment by kardon18** [comment]
I found the same thing happen.
The issue was also there in RC1. An attempt was made for a fix in RC2 . But the issue still remains.

The configurator can be closed and restarted, with the FC also being rebooted. Then the OSD page on the configurator will unlock, when Java script reloads.

**Comment by jreise-d** [comment]
Same here, blank OSD page in 2.1.1, regardless of inav version on my f411 wing. I can reproduce this on several computers with win7 or win10, if no inav configurator has run on it yet, or folder users\xxx\AppData\Local\inav-configurator doesn't exit.

My temporary solution is this: I start configurator **version 2.1.0**, connect f411 and open OSD page, everything seems to be ok. Then I try 2.1.1 and everything is ok, OSD page is working. It seems like version 2.1.0 is writing something into the folder \AppData\Local\inav-configurator that version 2.1.1 needs for proper function.

**Comment by noagendafpv** [comment]
No OSD issues for me, Matek F411-wing, loaded iNav 2.0.1 and the 2.1.1 config would not communicate. Tried with a second F411 and same result, I reverted to config 2.1.0 and no more problems.

**Comment by putimir** [comment]
Same here...

+: https://www.facebook.com/groups/1309125302467027/permalink/2074959035883646/

**Comment by putimir** [comment]
@JVN2000 can you rephrase issue title to be more descriptive, please

**Comment by machadofelipe** [comment]
Same issue here, both on Win10 and MacOS (both with Chrome version 71.0.3578.98 (Official Build) (64-bit)). I was coming from 2.0, and after installing 2.1.1, OSD page opens blank. By reverting back to 2.1.0, opening the OSD tab and them installing the 2.1.1 solves de issue.

**Comment by noagendafpv** [comment]
> No OSD issues for me, Matek F411-wing, loaded iNav 2.0.1 and the 2.1.1 config would not communicate. Tried with a second F411 and same result, I reverted to config 2.1.0 and no more problems.

I can actually change my above statement, the OSD tab seems to be what is locking up my configurator. I have gone back to 2.1.0 though and it seems to work better.

**Comment by MCGUIRKINATOR** [comment]
Still experencing the same issue with osd tab blank on 2.0.1 and 2.1.1

**Comment by Kennymartin** [comment]
same osd tab not working with latest configurator!  Any  fixes yet???????

**Comment by MCGUIRKINATOR** [comment]
I reflasehed a few times to 2.1.1 with a kakute f4 v2 and got mostly every
thing working gps compas baro and osd tab, but something  went wrong with
the i2c bus and baro and mag come and go and esc would not arm regardless
of protocol went back to betaflight got gps working but no mag or baro.
Must be a faulty board

**Comment by MCGUIRKINATOR** [comment]
Have you tried flashing your board to 2.1.0 with hex file from git hub that
had osd working. It took about 30 seconds to load the tab but I was able to
configure it.

**Comment by Stoeny** [comment]
If I go into the Font Manager and go back, I can configure the OSD as before.
Is it a Font Issue?

**Comment by DzikuVx** [comment]
This is a known bug that will be fixed. For now, for firmware family `2.0.x` use Configurator `2.0.x`.
Configurator `2.1.1` works only with `2.1` firmware

**Comment by Kennymartin** [comment]
Thank you for the timely reply Pawel & I look forward to the fix.
Martin.

**Comment by turbosi92** [comment]
I'm on the latest RC2 with 2.1.1 configurator on mac and the OSD will not load....FYI.  I read this as 2.1.1 works with latest 2.1 build but in my case it doesn't...at least on mac.

Older version of configurator on windows works fine.

**Comment by jreise-d** [comment]
As posted above, start **configurator 2.1.0** and open OSD page once, that helps. After that configurator 2.1.1 and it's OSD page works fine with iNav 2.0.x and 2.1.0

**Comment by lisdexia** [comment]
i am having the same issue with a mateksys f405 wing , the osd page is not loading inside the configurator however the osd is showing in the fpv feed, for the 2.0+ versions   however if i roll it all back to ver:1.9.1  it all works purfectly fine in the configurator and on the fpv osd feed.   
any news on a fix ?? id love to get flying.

**Comment by axelradd** [comment]
Same issue on Matek F722 SE (Blank screen)

**Comment by Frederx** [comment]
Having same issue with all F4 boards with 2.1.1 os Linux.

**Comment by cmichael261** [comment]
I was originally running a Matek f722std on inav 2.0.0 with no problem with the OSD or font editor. Today I upgraded to Inav 2.1.1 and the OSD tab in the configurator is blank. I can see a OSD display in my goggles but I think it is a default, I can not change it. 
So I guess I'm having the same issue as everyone else.

**Comment by mariniam** [comment]
same here with Matek f405 ctr. Empty OSD tab.

**Comment by Nyanadayo** [comment]
Likewise with a Matek F405 CTR. I have reverted back to v2.0.0 Configurator and to 2.0.0 Firmware, now the OSD is back. I had the same issue with v2.2 and v2.1 configurator. Noticed also that current Chrome App on Ubuntu also has an empty OSD Tab..if you need more info let me know.

**Comment by kd4kw** [comment]
What is strange is I tried to make some osd changes in cmd and it looked like they took but when I do a get osd its the same as before I tried to change them. I can rollback to any of the 1.9.x and the osd screen is fine with latest inav build

**Comment by Kennymartin** [comment]
Is it possible to adjust the OSD configuration via cli commands?
Im having a problem re-installing an earlier version of inav configurator on my 64bit windows machine and therefore cannot scale back my setup to adjust OSD settings via the configurator tab. 
I want to enable the home arrow & GPS coordinates so that they are visable via my googles as current default settings do not include such data. 
Any help would be appreciated.

**Comment by Kennymartin** [comment]
API version 2.20
FC Version 2.01

**Comment by DzikuVx** [comment]
@Kennymartin you can build you own from the code or wait for next release which should happen relatively soon

**Comment by Kennymartin** [comment]
No need to get bitchy now, you could of just said that in the first place.
Closing a thread issue in most peoples minds= fixed.

**Comment by shellixyz** [comment]
@Kennymartin  He is not bitchy he told you what you needed to know to make it work. Or course it is not going to work in the present version of the configurator the fix have been merged only 2 hours ago. It is effectively fixed. Just not available yet in builds. So just be a little patient.

**Comment by Kennymartin** [comment]
Thank you for the mannerly reply Michel.

**Comment by Painless360** [comment]
Thanks for all the hard work guys on getting rid of these little annoying bugs. As a community we're lucky to have you doing all of this work for free.. Happy flying!

**Comment by lisdexia** [comment]
YES AGREED , thankyou for your efforts. iNav is the best shit since cocaiahhhhhh sorry sliced bread
